resources/sdk/pureclouddotnet/templates/api_doc.mustache

```diff
@@ -51,7 +51,11 @@ namespace Example
             // Uncomment below to setup prefix (e.g. BEARER) for API key, if needed
             // Configuration.Default.ApiKeyPrefix.Add('{{{keyParamName}}}', 'BEARER');{{/isApiKey}}{{#isOAuth}}
             // Configure OAuth2 access token for authorization: {{{name}}}
-            Configuration.Default.AccessToken = "YOUR_ACCESS_TOKEN";{{/isOAuth}}{{/authMethods}}{{/hasAuthMethods}}
+            // The following example is using the [Authorization Code Grant](https://developer.mypurecloud.com/api/rest/authorization/use-authorization-code.html)
```

Comment:
Including the link to the docs is a nice touch, but this is in a code block so markdown won't be rendered here. It should just be plain text.